### PR TITLE
Add calendar

### PR DESCRIPTION
**Pull request by ghost**
[DEMO LINK](https://ruslan-pashkov.github.io/layout_calendar/)

### CODE REVIEW

File: src/styles/blocks/calendar.scss [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+  $days:
+    mon 0,
+    tue 1,
+    wed 2,
+    thu 3,
+    fri 4,
+    sat 5,
+    sun 6;
+
```
Comment (oleksandrnikora) [inline src/styles/blocks/calendar.scss:18]: It should move to variables file

File: src/styles/blocks/calendar.scss [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    + 2 * $calendar-padding;
+
+  $days:
+    mon 0,
+    tue 1,
+    wed 2,
+    thu 3,
+    fri 4,
+    sat 5,
+    sun 6;
```
Comment (oleksandrnikora) [inline src/styles/blocks/calendar.scss]: It should move to variables file

**Review by oleksandrnikora — APPROVED** [review]
Good job!

**Review by solaryasha — APPROVED** [review]
Good job!

**Pull request closed without merging**